cmd/snap/cmd_services.go

```diff
@@ -133,9 +134,13 @@ func (s *svcStatus) Execute(args []string) error {
 	w := tabWriter()
 	defer w.Flush()
 
-	fmt.Fprintln(w, i18n.G("Service\tStartup\tCurrent\tNotes"))
+	fmt.Fprintln(w, i18n.G("Service\tScope\tStartup\tCurrent\tNotes"))
```

Comment:
we need to decide whether to do this unconditionally or not?   we have the same issue/question for "snapctl services" as well which also will need changes

Reply:
I guess we could make an initial pass through the services and only include the column if we have any user daemons.  That would likely cover some use cases of people using `snap services` in a script, provided they limit the output to snaps they expect not to have user daemons.

I imagine `snapctl services` is probably more contentious, since it's unlikely that regular users would ever call it: outside of debugging, it's probably only used by scripts.  With that said, the "only include the scope column if there are user daemons" would probably be more effective given that it will only return results for the current snap.

Reply:
not a fan  of making the output variable like that:  " I guess we could make an initial pass through the services and only include the column"

As we discussed we can either try to check what's the fallback of going this way, or try to reuse the Notes column instead.  We do something similar for "snap list" and non-app snaps fwiw as it was mentioned.

Reply:
I've updated the pull request to add a "user" item to the existing notes column instead of adding a new column.  The `snapctl services` output has also been updated to match (including using a "-" for the active state of user services).